triggerbot/tree_watcher.py

```diff
@@ -96,15 +94,12 @@ def _get_jobs(self, branch, rev, hidden):
         return [job['ref_data_name'] for job in jobs
                 if not re.match('[a-z0-9]{12}', job['ref_data_name'])]
 
-
     def get_hidden_jobs(self, branch, rev):
         return self._get_jobs(branch, rev, True)
 
-
     def get_visible_jobs(self, branch, rev):
         return self._get_jobs(branch, rev, False)
 
-
```

Comment:
I see newlines being removed here, bringing us down to one line between methods, but added in dump_buildstats.py, bringing that file up to two lines between methods. Please make them consistent (I don't have a preference for one or two at this point, I think they're mixed right now).